### DISCUSSION TITLE
🎫不用付錢!就可以享受YouTube Premium

### DISCUSSION THREAD

**Discussion by JunTingLin** [1101-GS4719-C/Python-Community-GS4719-C#399] [General]
# 🎫不用付錢!就可以享受YouTube Premium

## 前言
[推薦好用的擋廣告app #395](https://github.com/NCU-GS4719-Python/Python-Community/discussions/395)，這篇同學推薦可以裝google插鍵Adblock for Youtube來阻擋youtube廣告，那是不是也希望手機看youtube片片時，也可以享受無廣告的福利😁，在此推薦YouTube Vanced這款app

## YouTube Premium 有啥好處❓
據我所知的有這幾個，也是我覺得最需要的

1. 看影片完全不會有靜態、動態廣告
2. 可以影片背景播放(也就說關閉螢幕還是可以有聲音) 

## YouTube Vanced app 安裝提醒
1. 這種app當然不可能在play商店給你找到啦，要透過apk安裝檔的方是安裝
2. 目前僅有Android

## 安裝教學

這邊就不依依細講啦!詳細請看:
[【APP推薦】YouTube Vanced安裝教學！讓你YouTube無廣告看影片+背景聽音樂！](https://www.jyes.com.tw/news.php?act=view&id=1002)

## 本人親測&提醒
我自己是有安裝並且使用的都非常順利，但還是要提醒各位，這app並是第三方軟體，登入google帳號時，可以用備用的信箱或是不要登入也可。

## apk載點
[官方網址](https://vancedapp.com/)
![螢幕擷取畫面 2022-03-04 001901](https://user-images.githubusercontent.com/92431095/156607024-1486829a-c699-49d0-92e5-23753cba180f.png)

**Comment by pekoranoinu** [comment]
感謝大佬的推薦，還幫我們親自實測。

**Comment by 11timyyy** [comment]
這讓我想到每次都會跳出惱人的premium廣告XD

**Comment by chupochun** [comment]
謝謝分享 這樣以後看影片就不需要等無聊又麻煩的廣告了告了

**Comment by yuchi06274** [comment]
謝謝分享!終於可以不用跳過廣告了

**Comment by Yuhsin000** [comment]
不過premium下載影片很方便，有離線需求的還是可以買一下，單純聽音樂可以查youtube轉mp3，可以下載聆聽~

**Comment by PoooYen** [comment]
mac ios的已哭
